src/TicketService.java

```diff
@@ -1,37 +1,74 @@
-import java.util.Calendar;
+import java.util.*;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
 
 public class TicketService {
-    public static void main(String[] args) {
-        try {
-            Ticket emptyTicket = new Ticket();
-            System.out.println(emptyTicket);
-
-            Calendar calendar = Calendar.getInstance();
-            calendar.set(2024, Calendar.AUGUST, 15, 12, 0, 0);
-            calendar.set(Calendar.MILLISECOND, 0);
-            long eventTime = calendar.getTimeInMillis();
-
-            Ticket fullTicket = new Ticket("ID01",
-                    "Concert",
-                    "001", eventTime,
-                    true,
-                    'A',
-                    5.5);
-            System.out.println(fullTicket);
-
-            calendar.set(2024, Calendar.AUGUST, 30, 16, 2, 0);
-            long eventTime1 = calendar.getTimeInMillis();
-
-            Ticket limitedTicket = new Ticket("Hall",
-                    "002",
-                    eventTime1);
-            System.out.println(limitedTicket);
-
-            //saving ticket price
-            emptyTicket.toSaveTicketPrice(49.99);
-
-        } catch (IllegalArgumentException e) {
-            System.out.println(e.getMessage());
-        }
-    }
+	public static void main(String[] args) {
+		try {
+			Ticket emptyTicket = new Ticket();
+			System.out.println(emptyTicket);
+
+			Calendar calendar = Calendar.getInstance();
+			calendar.set(2024, Calendar.AUGUST, 15, 12, 0, 0);
+			calendar.set(Calendar.MILLISECOND, 0);
+			long eventTime = calendar.getTimeInMillis();
+
+			Ticket fullTicket = new Ticket("ID01",
+					"Concert",
+					"001", eventTime,
+					true,
+					'A',
+					5.5);
+			System.out.println(fullTicket);
+
+			calendar.set(2024, Calendar.AUGUST, 30, 16, 2, 0);
+			long eventTime1 = calendar.getTimeInMillis();
+
+			Ticket limitedTicket = new Ticket("Hall",
+					"002",
+					eventTime1);
+			System.out.println(limitedTicket);
+
+			//saving ticket price
+			emptyTicket.toSaveTicketPrice(49.99);
+
+		} catch (IllegalArgumentException e) {
+			System.out.println(e.getMessage());
+		}
```

Comment:
just a question - in which case it could be?

Reply:
When incorrect Ticket's arguments would be provided.  I throw new IllegalArgumentException with messages in Ticket class's sets.
P.S. Was it a question to me? :)